code/modules/events/mimite_infestation.dm

```diff
@@ -58,7 +58,7 @@
 
 
 /datum/round_event/mimite_infestation/end()
-	if(LAZYLEN(GLOB.all_mimites) >= 1) //If they're still around at the end of 60 min, turn of replication permanently
+	if(length(GLOB.all_mimites)) //If they're still around at the end of 60 min, turn of replication permanently
```

Comment:
If all mimites is a lazy list, then it should be using LAZYLEN. If it isn't, then you shouldn't be using LAZYADD

Reply:
Tis a list, was suggested to do this here - https://github.com/BeeStation/BeeStation-Hornet/pull/11194#discussion_r1682049951

Reply:
will revert